### DISCUSSION TITLE
Removing a specific service resources that is located in the same directory as other services yaml files

### DISCUSSION THREAD

**Discussion by jovanvuleta** [serverless/serverless#10738] [Q&A] [answered]
- After reading the official documentation I saw that in order to delete deployed services resources on AWS I have to navigate to the directory of the services YAML file location and run "**serverless remove**".(https://www.serverless.com/framework/docs/providers/aws/cli-reference/remove)
- The problem with that approach in my case is that I have 5 YAML configs(therefore 5 services) in the same directory. (Under **/project** path are located all 5 YAML files and each service has other resources deployed)
So running **"serverless remove"** doesn't specify which exact service is going to be deleted since all of them are located under path **"/project"**.
- Btw all the services are in the same region and the environment names are the same as in the other services so going with the options "**--region"** or "**--stage**" I don't see helping in this case.

If anyone knows how can I specify the exact service to be deleted in that directory where all 5 YAML files are located I would appreciate it...

**Comment by pgrzesik** [comment, marked as answer]
Hello @jovanvuleta - if you have multiple configurations, you can specify config to be used with `--config <path-to-config>` flag, same as for `deploy`. 

For example:

```
serverless remove --config serverless.api.yml
serverless remove --config serverless.resources.yml
```

and so on.

**Reply by jovanvuleta** [reply]
Works, thank you!🙌🏻
